bin/check-wsrep-ready.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+#
+# Check Cluster Status Plugin
+# ===
+#
+# This plugin checks the wsrep_ready status of the cluster.
+#
+# Copyright 2016 Antonio Berrios aberrios@psiclik.plus.com
+
+# Based on the Percona Cluster Size Plugin by Chris Alexander <chris.alexander@import.io>, import.io; which is based on
+# Based on the MySQL Health Plugin by Panagiotis Papadomitsos
+#
+# Depends on mysql:
+# gem install mysql
+#
+# Released under the same terms as Sensu (the MIT license); see LICENSE
+# for details.
+
```

Comment:
can you use the standard header from http://sensu-plugins.io/docs/developer_guidelines.html